### PR TITLE
Reconcile ChainID changes for Sandboxnet

### PR DESCRIPTION
**Pull request by sjonpaulbrown**
## Description
The update of the Stagingnet chain name was added to the `v0.27` branch, but it was never merged to master. We need to reconcile the changes and get these merged to the `v0.28` branch.

### CODE REVIEW

File: cmd/scaffold.go [hunk 1]
```diff
@@ -946,7 +946,16 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
...
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
-		fvm.WithTransactionFeesEnabled(true),
+	}
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Sandboxnet || fnb.RootChainID == flow.Mainnet {
+		vmOpts = append(vmOpts,
+			fvm.WithTransactionFeesEnabled(true),
+		)
+	}
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Mainnet {
```
Comment (pattyshack) [inline cmd/scaffold.go:955]: lint issue: undefined value

also, combine the two if statements.

**Review by vishalchangrani — APPROVED** [review]

File: cmd/scaffold.go (hunk 1 above)
Comment (sjonpaulbrown) [reply cmd/scaffold.go:955]: I will take a look at this. It looks like I messed up fixing a conflict.

File: cmd/scaffold.go [hunk 2]
```diff
@@ -946,9 +946,13 @@ func (fnb *FlowNodeBuilder) initFvmOptions() {
 		fvm.WithChain(fnb.RootChainID.Chain()),
 		fvm.WithBlocks(blockFinder),
 		fvm.WithAccountStorageLimit(true),
-		fvm.WithTransactionFeesEnabled(true),
 	}
-	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Stagingnet || fnb.RootChainID == flow.Localnet || fnb.RootChainID == flow.Benchnet {
+	if fnb.RootChainID == flow.Testnet || fnb.RootChainID == flow.Sandboxnet || fnb.RootChainID == flow.Mainnet {
```
Comment (janezpodhostnik) [inline cmd/scaffold.go:950]: Its weird that benchnet wouldn't have fees, but I guess this is just a port.

**Review by janezpodhostnik — APPROVED** [review]

File: cmd/scaffold.go (hunk 2 above)
Comment (janezpodhostnik) [reply cmd/scaffold.go:950]: cc: @SaveTheRbtz just for visibility

**Review by pattyshack — APPROVED** [review]

**Pull request merged**